.github/workflows/build-linux-installer-deb.yml

```diff
@@ -212,6 +197,80 @@ jobs:
         name: chia-installers-linux-deb-${{ matrix.os.arch }}
         path: ${{ github.workspace }}/build_scripts/final_installer/
 
+    - name: Remove working files to exclude from cache
+      run: |
+        rm -rf ./chia-blockchain-gui/packages/gui/daemon
+
+  publish:
+    name: Publish ${{ matrix.os.arch }}
+    runs-on: ${{ matrix.os.runs-on }}
+    needs:
+    - build
+    container: chianetwork/ubuntu-20.04-builder:latest
+    timeout-minutes: 5
+    strategy:
+      fail-fast: false
+      max-parallel: 4
```

Comment:
```suggestion
```
Assume this was a copy/paste, but generally we dont need to limit matrices in cases like this (Looking earlier in the file, I now see this was already there for the other matrix.. might make sense to just get rid of the max-parallel for all the installer workflows)